internal/core/entities/number.go

```diff
@@ -9,7 +9,7 @@ import (
 type Number struct {
 	DBModel
 	No               string    `gorm:"Column:number" json:"number"`
-	UserId           uuid.UUID `gorm:"type:uuid" json:"user_id"`
+	UserId           uuid.UUID `gorm:"Column:userid;type:uuid" json:"user_id"`
```

Comment:
user_id is the best practice we tend to separate different words with underscore in databases and json.